geopackage_validator/output.py

```diff
@@ -1,9 +1,26 @@
 import json
+import yaml
+from collections import OrderedDict
 from datetime import datetime
 from typing import Dict, List
+from collections import OrderedDict
+
 from geopackage_validator import __version__
 
-import pkg_resources  # part of setuptools
+
+def represent_ordereddict(dumper, data):
```

Comment:
FYI dit is een manier om 
```suggestion
def represent_ordereddict(dumper, data):
"""
Represent method for pyyaml. 
Shamelessly copied from the python3 part of this answer: https://stackoverflow.com/questions/16782112/can-pyyaml-dump-dict-items-in-non-alphabetical-order#answer-16782282"""
```